installing/README.md

```diff
@@ -0,0 +1,10 @@
+# Installing Nix
```

Comment:
Well i would put <10 lines on a separate page as an intro explaining what it is, otherwise why would i continue reading if i don't even know what it is (like the intro page of http://rustbyexample.com/, but with a little bit more detail since it's not as simple as a language)
Also the wording makes it sound like it is a language specific package manager